wheel_fl:
    controller:
        package: dynamixel_controllers
        module: joint_torque_controller
        type: JointTorqueController
    joint_name: fl
    joint_speed: 1.0
    motor:
        id: 1
        init: 0
        min: 0
        max: 4093

wheel_fr:
    controller:
        package: dynamixel_controllers
        module: joint_torque_controller
        type: JointTorqueController
    joint_name: fr
    joint_speed: 1.0
    motor:
        id: 2
        init: 0 
        min: 0
        max: 4093


pos_fl:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: joint_fl
    joint_speed: 1023 
    motor:
        id: 1
        init: 512 
        min: 0
        max: 1023

pos_fr:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: joint_fr
    joint_speed: 1023 
    motor:
        id: 2
        init: 512 
        min: 0
        max: 1023
